code.js

```diff
@@ -1,7 +1,11 @@
 function sum(a) {
-    var sum = a[0];
+    var sum = 0;
     for(var i = 0; i < a.length; i++) {
         sum += a[i];
     }
     return sum;
 }
+
+let testArray = [5, 10, 15]
```

Comment:
Please don't leave the debug code in your submission.

Reply:
Okay, I removed all of it and then re-committed.